ui/Dockerfile.debug

```diff
@@ -0,0 +1,40 @@
+# This dockerfile is for development purposes only; do not use this for production deployments
```

Comment:
An alternative to this file would be to symlink the license file into the ui folder.

I chose this path because the symlink would be likely to be unintentionally deleted due to perceived redundancy, and the generated file won't contain the required notice.